packages/core/src/Stores/client-store.js

```diff
@@ -1724,7 +1725,7 @@ export default class ClientStore extends BaseStore {
                 this.setAccountSettings((await WS.authorized.cache.getSettings()).get_settings);
             }
 
-            if (this.account_settings) this.setPreferredLanguage(this.account_settings.preferred_language);
+            if (this.account_settings) this.setPreferredLanguage(this.account_settings?.preferred_language);
```

Comment:
why do we need optional chaining here? This code block will be executed only if we have  account_settings. (line 1728)